### ISSUE TITLE
vocoder 1337 - doc

### ISSUE THREAD

**Issue by trebmuh** [swh/ladspa#30]
Please, it would be great to add a description of it on http://plugin.org.uk/ladspa-swh/docs/ladspa-swh.html (keeping consistency up to date).

**Comment by tresf** [comment]
@swh can you please add the vocoder instructions to your HTML?

http://lmms.io/wiki/index.php?title=Working_with_LADSPA_effects#Vocoder

**Comment by tresf** [comment]
Hmm... The LMMS instructions are too wordy and too specific to LMMS.

Here's my version.  Feel free to modify... 

> Vocoder or "Voice Encoder" can create a pitched voice based on a second input signal.
 * To use this plugin, pass two tracks through it at once
  * Pan the voice to the full left
  * Pan the carrier wave to the full right

> The output is a vocoded voice track based on the carrier wave.  Hint: Use saw wave, NOT sign wave as right channel must contain contain a broad wave-spectrum, with several overtones.

**Comment by swh** [comment]
The docs are generated automatically from the XML source, see https://github.com/swh/ladspa/blob/master/docs/Makefile

Unfortunately I don't have a current build system that can run them, my hosting account used to have all the needed tools built, but it's so long since I ran it that the binaries won't execute anymore.

I'll have a go at getting a docs build environment working at some point, but it's likely to be a couple of months - I'm out of the country for most of June. It needs some pretty ancient software.

**Comment by jasp00** [comment]
@trebmuh, @tresf, I am not sure this is an issue. http://plugin.org.uk/ladspa-swh/docs/ladspa-swh.html is consistent because http://plugin.org.uk/ladspa-swh/ does not have vocoder. The documentation is autogenerated from the XML files. In GitHub, vocoder already has a basic description:

> Makes a human voice sound synthetic; often used to speak like a robot, with a metallic and monotonous voice.  Using a human voice as the formant and an instrument as the carrier, creates an effect of an instrument speaking.

Panning to left and right channels is specific to LMMS; the plug-in has `Formant-in` and `Carrier-in` inputs. Perhaps the hint about "use saw wave, not sine wave" could be added.

**Comment by tresf** [comment]
@jasp00 good points.  Can do the documentation PR?

**Comment by jasp00** [comment]
@tresf, done in #31.

**Comment by trebmuh** [comment]
Awesome. Thanks to you all.
